app/browser/api/ledger.js

```diff
@@ -74,6 +74,7 @@ const v2RulesetPath = 'ledger-rulesV2.leveldb'
 let v2PublishersDB
 const v2PublishersPath = 'ledger-publishersV2.leveldb'
 const statePath = 'ledger-state.json'
+const newClientPath = 'ledger-newstate.json'
```

Comment:
Can we change this to `const newClientPath = pathName('ledger-newstate.json')`, so that we call `pathName` only once. Same can be done with `statePath`

Reply:
I looked at doing this, but since it uses `electron.app.getPath()` (which may or may not be initialized), I think changing that may be better to do in a different PR. Thanks for the suggestion though 😄